### PR TITLE
Report loaded extensions

### PR DESCRIPTION
**Pull request by kesselb**
Include the loaded extensions in usage survey.

Motivation: Evaluate the removal of the php-intl polyfill.

### CODE REVIEW

**Comment by nickvergessen** [comment]
> Motivation: Evaluate the removal of the php-intl polyfill.

I'm not sure this even helps.

Polyfill should be unrelated? 

But anyway, I guess it's fine.

File: lib/Categories/Php.php [hunk 1]
```diff
@@ -69,6 +69,7 @@ public function getData() {
 			'memory_limit' => $this->phpIni->getBytes('memory_limit'),
 			'max_execution_time' => $this->phpIni->getNumeric('max_execution_time'),
 			'upload_max_filesize' => $this->phpIni->getBytes('upload_max_filesize'),
+			'loaded_extensions' => implode(',', get_loaded_extensions() ?? [])
```
Comment (nickvergessen) [inline lib/Categories/Php.php]: Did you check with a local version, that the survey_server app doesn't crash with the value missing/being there?
Also you need to add it to https://github.com/nextcloud/survey_server/blob/master/data.json

Comment (kesselb) [reply lib/Categories/Php.php]: Not yet :see_no_evil: Thanks for explaining :+1:

**Comment by kesselb** [comment]
> > Motivation: Evaluate the removal of the php-intl polyfill.
> 
> I'm not sure this even helps.
> 
> Polyfill should be unrelated?
> 
> But anyway, I guess it's fine.

Hmm. I would like to know how many instances (which participate in the usage survey) have the native intl extension available.

**Comment by nickvergessen** [comment]
I just noticed now that it is only listed in recommended in the manual setup,
But the example does install it, so I guess most users will have it:
https://docs.nextcloud.com/server/stable/admin_manual/installation/source_installation.html#example-installation-on-ubuntu-18-04-lts-server

**Comment by kesselb** [comment]
> so I guess most users will have it

I think so too because without there is this [setup check](https://github.com/nextcloud/server/blob/61661036fc986f908fe975d52b8b0cdcf5eb2272/apps/settings/lib/Controller/CheckSetupController.php#L570-L572) and a [annoying log message](https://github.com/nextcloud/3rdparty/blob/4a0dc8066bde4ca5641a740e304f9a5403a5952e/patchwork/utf8/src/Patchwork/Utf8/Bootup/intl.php#L18). But we don't know for sure ;) I'm fine with closing this pr if this is too much work.

**Comment by nickvergessen** [comment]
Let's close it, and instead change the docs to always install the extension

**Pull request closed without merging**